tools/CreateMappings_rules.json

```diff
@@ -134,5 +134,6 @@
 
     { "regex": "ManagedServiceIdentity", "group": "Managed Service Identity", "alias": "Managed Service Identity" },
 
-    { "regex":  "ManagementPartner", "alias":  "Management Partner"  }
+    { "regex":  "ManagementPartner", "alias":  "Management Partner"  },
+    { "regex": "DevSpaces", "group": "DevSpaces", "alias": "DevSpaces" }
 ]
```

Comment:
You'll need to remove the two packages below before releasing, once your SDK is published (commenting for me to remember in the future).

Reply:
N/A